templates/core/about/builds.html

```diff
@@ -7,72 +7,76 @@
 	<h1>Builds</h1>
 	<div class="container about">
 	<p>
-		Docs.rs automatically builds crates' documentation released on
-		<a href="https://crates.io/">crates.io</a>
-		using the nightly release of the Rust compiler.
-		Builds can take a while depending how many crates are in <a href="/releases/queue">the queue</a>.
+		Docs.rs automatically builds documentation for crates released on
+		<a href="https://crates.io/">crates.io</a>.
+		 
+		It may take a while to build your crate depending on how many crates are in <a href="/releases/queue">the queue</a>.
 	</p>
 
 	<p>
+		All crates are build in a sandbox using the nightly release of the Rust compiler.
 		{%- if rustc_version %}
-		The current version of the Rust compiler in use is <code>{{ rustc_version }}</code>.
+		The current version in use is <code>{{ rustc_version }}</code>.
```

Comment:
```suggestion
		The current version of <code>rustc</code> in use is <code>{{ rustc_version }}</code>.
```

Reply:
This may not be needed, @jyn514 I don't remember the format that `rustc_version` takes, either just a version & hash or the full thing

Reply:
You can see it on the website: 
![image](https://user-images.githubusercontent.com/23638587/89739596-e6f04900-da4f-11ea-96b6-b8196bcf5d5a.png)

Reply:
Currently the string emitted by `rustc_version` is "rustc 1.47.0-nightly (6c8927b0c 2020-07-26)". Thus adding another "rustc" would be redundant.